network/wrappers/identity.go

```diff
@@ -0,0 +1,58 @@
+//nolint:dupl
+package wrappers
+
+import (
+	"context"
+
+	"github.com/dogechain-lab/dogechain/network/proto"
+	"github.com/hashicorp/go-hclog"
+	"go.uber.org/atomic"
+
+	rawGrpc "google.golang.org/grpc"
+)
+
+type IdentityClient interface {
+	Hello(ctx context.Context, in *proto.Status) (*proto.Status, error)
+
+	GrpcClientWrapper
```

Comment:
It is common to put embedded fields first.

Reply:
done